logging/logging.go

```diff
@@ -167,11 +175,15 @@ func NewClient(ctx context.Context, parent string, opts ...option.ClientOption)
 	return client, nil
 }
 
-func makeParent(parent string) string {
+func makeParent(parent string) (string, error) {
 	if !strings.ContainsRune(parent, '/') {
-		return "projects/" + parent
+		return "projects/" + parent, nil
 	}
-	return parent
+	prefix := strings.Split(parent, "/")[0]
+	if prefix != "projects" && prefix != "folders" && prefix != "billingAccounts" && prefix != "organizations" {
+		return parent, fmt.Errorf("Parent parameter should start with 'projects/', 'folders/', 'billingAccounts/' or 'organizations/'")
```

Comment:
Errors should start with a lowercase letter. Remove all comma delimitation and just put spaces between different resources path prefixes.